libraries/helpers.rb

```diff
@@ -110,6 +110,17 @@ def version_latest?(vers)
     end
 
     #
+    # Returns the custom setup recipe
+    #
+    # When the user specifies this attribute chef-ingredient will not configure
+    # our default packagecloud Chef repositories and instead it will include the
+    # custom recipe. This will eliminate the hard dependency to the internet.
+    #
+    def custom_repo_setup_recipe
+      node['chef-ingredient']['custom_repo_setup_recipe'] if node['chef-ingredient']
+    end
```

Comment:
I think this should be a predicate method, so when we encounter it in the code in six months we know what that's about. Second, we don't need the `if` clause because the attributes file will have the `'chef-ingredient'` attribute. Third, I think we should use the `attribute?` predicate.

```ruby
def use_custom_repo_recipe?
  node['chef-ingredient'].attribute?('custom-repo-recipe')
end
```

Then in the places where the method was used with the implicit return of the attribute value, we use the attribute directly. If we really want to use it in a method, we could do:

```ruby
```